src/welcome/services.ts

```diff
@@ -1,137 +1,62 @@
-import * as fs from "fs";
-import helpers from "../../shared/helpers";
-import { BrowserWindow } from "electron";
+import * as fs from 'fs'
```

Comment:
Maybe only import the functions from `fs` that are being used in this class?

Reply:
Okay